src/gtk-sat-module-popup.c

```diff
@@ -1261,10 +1261,11 @@ gboolean module_window_config_cb(GtkWidget * widget, GdkEventConfigure * event,
 #ifdef G_OS_WIN32
     /* Workaround for GTK+ bug # 169811 - "configure_event" is fired
        when the window is being maximized */
+    /* PAA: going to assume this is fixed in GTK-3...
     if (gdk_window_get_state(widget->window) & GDK_WINDOW_STATE_MAXIMIZED)
     {
         return FALSE;
-    }
+    } */
```

Comment:
The `GtkWidget` in Gtk3 does *not* have a `windows` member. So shouldn't it simply say:
``` 
if (gdk_window_get_state(gtk_widget_get_window(widget)) & GDK_WINDOW_STATE_MAXIMIZED)
```

Reply:
Very probably, if needed at all since this is a workaround for a Gtk 2 bug. I'll check the bug report and update as appropriate. Thanks for the review.

Reply:
Another idea I have; why not *require* libcurl and use that for the Hamlib connection too?  Like a:
` curl_easy_setopt(curl, CURLOPT_CONNECT_ONLY, 1)`. etc.
I don't know the details of Hamlib. Maybe it's close to a *ping-pong* protocol. Then `libcurl` could be used for *all* Hamlib comms.